lib/wow_data/auth/db.ex

```diff
@@ -0,0 +1,30 @@
+defmodule WoWData.Auth.Db do
+  alias WoWData.Repo
+
+  def create(schema, module, attrs) do
```

Comment:
Adiciona @spec nas funções publicas.

Reply:
https://elixir-lang.org/getting-started/typespecs-and-behaviours.html#types-and-specs

Reply:
Beleza, também vou adicionar uma documentação mais detalhada e tentar deixar parecida com a doc de cobrança